code/modules/vehicles/ridden.dm

```diff
@@ -0,0 +1,42 @@
+
+/obj/vehicle/ridden
+	name = "ridden vehicle"
+	can_buckle = TRUE
+	max_buckled_mobs = 1
+	buckle_lying = FALSE
+	var/key_required = FALSE	//ONLY FOR INSERTED KEYS. FOR HELD KEYS, SPECIFY IN RIDING DATUM'S KEYCHECK.

```

Comment:
why is this on ridden level and why dont you just check if(key_type) instead of having 2 vars (key_required and key_type)

Reply:
fixed

Reply:
.pet kevinz000